GitUI/CommandsDialogs/SettingsDialog/Pages/SshSettingsPage.cs

```diff
@@ -15,7 +16,7 @@ public partial class SshSettingsPage : SettingsPageWithHeader
         public SshSettingsPage()
         {
             InitializeComponent();
-            Text = "SSH";
+            Text = GitProtocol.Ssh.ToString("G");
```

Comment:
What does this do?

Reply:
I searched for the "ssh" key when I refactored and I hesitated to replace it.
Thinking about it again, it was a bad idea. I will revert it.

Reply:
For an enum, to string return the int value as string and `.ToString("G")` return the name of the key, so here "Ssh".